pennylane/operation.py

```diff
@@ -164,6 +164,12 @@ class ObservableReturnTypes(Enum):
 """Enum: An enumeration which represents returning probabilities
 of all computational basis states."""
 
+_measurement_type_map = {
+    Expectation: "expval",
+    Variance: "var",
+    Sample: "sample",
+}
```

Comment:
I'm wondering if it's possible to define the `__repr__` for an enum? That way it can be done in the enum directly.

Reply:
It is, do you think it'd look nicer?
You could even get rid of the dict if you changed the names of the enum members to expval, var, sample etc.

Reply:
The capital in the name was chosen for convention (based on the python Enum docs), I worry changing it to lowercase might cause ambiguities to people reading the code. But I'm on the fence!

Reply:
How about this, using the strings as the enum values?

Reply:
Nice! I didn't know `repr` was a function :thinking: